dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -587,12 +603,12 @@ will be generating a library in it.  Let's add the argument MYEXTLIB to
 the WriteMakefile call so that it looks like this:
 
 	WriteMakefile(
-	    'NAME'      => 'Mytest2',
-	    'VERSION_FROM' => 'Mytest2.pm', # finds $VERSION
-	    'LIBS'      => [''],   # e.g., '-lm'
-	    'DEFINE'    => '',     # e.g., '-DHAVE_SOMETHING'
-	    'INC'       => '',     # e.g., '-I/usr/include/other'
-	    'MYEXTLIB' => 'mylib/libmylib$(LIB_EXT)',
+	    NAME         => 'Mytest2',
+	    VERSION_FROM => 'Mytest2.pm', # finds $VERSION
+	    LIBS         => [''], # e.g., '-lm'
+	    DEFINE       => '', # e.g., '-DHAVE_SOMETHING'
+	    INC          => '-I.', # e.g., '-I. -I/usr/include/other'
```

Comment:
Nor this.

Reply:
Resolved as the above comment.

Reply:
EXAMPLE 4

The tutorial instructs to add in Makefile.PL an argument to the WriteMakefile call and to add a subroutine.

I take the presumption here that further editing of the WriteMakefile call would perhaps oblige justification.

Reply:
*all* changes oblige justification, and should be grouped into separate pull requests when unrelated.